app/models/health/exporter/health_exporter.rb

```diff
@@ -0,0 +1,222 @@
+      ssms: {
+        file_name: 'self_sufficiency_matrix_forms.csv',
+        columns: [:id, :patient_id, :completed_at, :created_at, :updated_at],
+        ids: {
+          id: :ssm_id,
+          patient_id: :patient_id,
+        },
+        model_class: Health::SelfSufficiencyMatrixForm,
+        association_id: :patient_id,
+      },
+      participation_forms: {
+        file_name: 'participation_forms.csv',
+        columns: [:id, :patient_id, :signature_on, :location, :reviewed_at, :reviewer],
+        ids: {
+          id: :participation_id,
+          patient_id: :patient_id,
+        },
+        model_class: Health::ParticipationForm,
+        association_id: :patient_id,
+      },
+      chas: {
+        file_name: 'comprehensive_health_assessments.csv',
+        columns: [:id, :patient_id, :status, :created_at, :reviewed_at, :completed_at, :reviewed_at, :reviewer],
+        ids: {
+          id: :cha_id,
+          patient_id: :patient_id,
+        },
+        model_class: Health::ComprehensiveHealthAssessment,
+        association_id: :patient_id,
+      },
+      careplans: {
+        file_name: 'careplans.csv',
+        columns: [:id, :patient_id, :created_at, :updated_at, :patient_signed_on, :provider_signed_on, :status],
+        ids: {
+          id: :careplan_id,
+          patient_id: :patient_id,
+        },
+        model_class: Health::Careplan,
+        association_id: :patient_id,
+      },
+      qas: {
+        file_name: 'qualifying_activities.csv',
+        columns: [:id, :mode_of_contact, :mode_of_contact_other, :reached_client, :reached_client_collateral_contact,
+          :activity, :date_of_activity, :patient_id, :created_at, :updated_at, :naturally_payable, :duplicate_id,
+          :valid_unpayable, :procedure_valid],
+        ids: {
+          id: :qa_id,
+          patient_id: :patient_id,
+        },
+        model_class: Health::QualifyingActivity,
+        association_id: :patient_id,
+      },
+    }
+
+    FAKERS = {
+      id_in_source: [ Faker::IDNumber, :valid ],
+      first_name: [ Faker::Name, :first_name ],
+      last_name: [ Faker::Name, :last_name ],
+      medicaid_id: [ Faker::Number, :number, {digits: 12} ],
+      birthdate: [ Faker::Date, :birthday ],
+      completed_at: :keep,
+      ssn: [Faker::IDNumber, :valid],
+      created_at: :keep,
+      updated_at: :keep,
+      engagement_date: :keep,
+      coverage_level: :keep,
+      coverage_inquiry_date: :keep,
+      aco_name: [ Faker::Company, :name ],
+      previous_aco_name: [ Faker::Company, :name ],
+      invalid_id: :keep,
+      enrollment_start_date: :keep,
+      disenrollment_date: :keep,
+      stop_reason_description: :keep,
+      pending_disenrollment_date: :keep,
+      contributing: :keep,
+      current: :keep,
+      rejected: :keep,
+      rejected_reason: :keep,
+      mode_of_contact: :keep,
+      mode_of_contact_other: :keep,
+      reached_client: :keep,
+      reached_client_collateral_contact: :keep,
```

Comment:
I think `reached_client_collateral_contact` should be faked, it's a free-form string